libraries/AC_PID/AC_PID.cpp

```diff
@@ -271,7 +271,8 @@ float AC_PID::get_ff()
 
 void AC_PID::reset_I()
 {
-    _integrator = 0.0;
+    _integrator = 0.0f;
```

Comment:
We actually don't need the `f`, the compiler can tell its supposed to be a float.